volontulo/views/organizations.py

```diff
@@ -44,8 +45,24 @@ def get(request):
         )
 
     @staticmethod
+    @login_required
     def post(request):
         u"""Method responsible for saving new organization."""
+        if not (
+                request.POST.get('name') and
+                request.POST.get('address') and
+                request.POST.get('description')
+        ):
+            yield_message_error(
+                request,
+                u"Należy wypełnić wszystkie pola formularza."
+            )
+            return render(
+                request,
+                "organizations/organization_form.html",
+                {'organization': Organization()}
```

Comment:
OK, we'e checked template and it should work right now. In future we will go with coherent solution with other pages.